src/components/hr/hr.component.tsx

```diff
@@ -19,6 +19,9 @@ export const Hr = ({
   const largeScreen = useIsAboveBreakpoint(adaptiveMxBreakpoint);
   let marginLeft = ml;
   let marginRight = mr;
+  // We have had to ignore the coverage for this if statement as it is not possible to test the adaptive behaviour in RTL.
```

Comment:
Overall comment is fine but I think we should include that the issue is due to JSDOM and how toHaveStyle works etc
```
// Coverage has been ignored as JSDOM does not implement `matchMedia` meaning the computed styles asserted against via `toHaveStyle` will fail
// A Playwright test has been added to catch any regressions